index.html

```diff
@@ -66,33 +78,40 @@ <h2 class="Description1">I'am a Software Developer</h2>
             ><img src="images/Icon - Twitter.png" alt="Twitter icon"
           /></a>
         </li>
+        <li class="medium">
+          <a href=""
+            ><img src="images/medium.svg" alt="Medium icon"
+          /></a>
+        </li>
       </ul>
     </section>
     <section class="grid-container">
       <div class="main-div">
         <div class="top-div">
           <h2 id="MyRecentWorks">My Recent Works</h2>
           <img src="images/Vector.png" alt="horizontal-line" id="hr-line" />
+        </div>
```

Comment:
- [ ] You did a great job 😃 on the project, but I see that the align of text here on the page not aligning with the cards, so please try to fix that to make it like the Figma Design 💯😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/230615724-cf6f53b6-e6d5-4129-9988-2224f8abcd73.png)   |  ![image](https://user-images.githubusercontent.com/83360636/230615689-22679126-2b44-4ee7-8a05-2d9ed1fcdf26.png) |